Add notes about volume sizes before Pulp3 migration (#931)
theforeman/foreman-documentation: guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc

To prevent this, pre-migrate Pulp content while running the latest version of {ProjectServer} {ProjectVersionPrevious}.
This reduces the overall upgrade downtime.

> During migration to Pulp3, amount of data stored in the `/var/lib/pulp/published/` directory can be doubled.

Reviewer: ```suggestion
During migration to Pulp3, the amount of data stored in the `/var/lib/pulp/published/` directory can double in size.
```

> Ensure there is enough space on the `/var/lib/pulp` volume.

Reviewer: ```suggestion
Ensure that there is enough space on the `/var/lib/pulp` volume.
```
```suggestion
Ensure there is enough space on the `/var/lib/pulp` volume.
```

> If the user has run `{foreman-maintain} prep-6.10-upgrade`, the content in `/var/lib/pulp/content` should not need to be duplicated.

Reviewer: ```suggestion
If the user runs `{foreman-maintain} prep-6.10-upgrade`, the content in `/var/lib/pulp/content` does not need to be duplicated.
```

> After Pulp2 is removed, the extra space can be claimed back, however, shrinking of XFS volumes is not possible and copying data to a different partition might be necessary in order to bring volume size down to the previous value.

Reviewer: ```suggestion
After Pulp2 is removed, the extra space can be claimed back, however, shrinking of XFS volumes is not possible and copying data to a different partition might be necessary in order to bring volume size down to the previous value.
```

> During migration to Pulp3, amount of data stored in the PostgreSQL data directory can grow significantly.

Reviewer: ```suggestion
During migration to Pulp 3, the amount of data stored in the PostgreSQL data directory can grow significantly.
```

> Make sure to have enough space for the migration.

Reviewer: ```suggestion
Ensure that you have enough space for the migration.
```

> Rough estimation is 2-3 times the size of the MongoDB Pulp2 database.

Reviewer: ```suggestion
A rough estimation of the space you require is 2-3 times the size of the MongoDB Pulp 2 database.
```
